modules/meb_api/spec/dgi/forms/service/toe_submission_service_spec.rb

```diff
@@ -68,19 +68,60 @@
       }
     end
 
+    let(:dd_params_lighthouse) do
+      {
+        paymentAccount: {
+          accountType: 'CHECKING',
+          accountNumber: '1234567890',
+          financialInstitutionRoutingNumber: '031000503',
+          financialInstitutionName: 'WELLSFARGO BANK'
+        },
+        controlInformation: {
+          canUpdateDirectDeposit: true,
+          isCorpAvailable: true,
+          isCorpRecFound: true,
+          hasNoBdnPayments: true,
+          hasIndentity: true,
+          hasIndex: true,
+          isCompetent: true,
+          hasMailingAddress: true,
+          hasNoFiduciaryAssigned: true,
+          isNotDeceased: true,
+          hasPaymentAddress: true,
+          isEduClaimAvailable: true
+        }
+      }
+    end
+
     describe '#submit_toe_claim' do
       let(:faraday_response) { double('faraday_connection') }
 
       before do
         allow(faraday_response).to receive(:env)
       end
 
-      context 'when successful' do
+      context 'EVSS when successful' do
         it 'returns a status of 200' do
           VCR.use_cassette('dgi/forms/submit_toe_claim') do
+            Flipper.disable(:toe_light_house_dgi_direct_deposit)
+
             response = service.submit_claim(ActionController::Parameters.new(claimant_params),
                                             ActionController::Parameters.new(dd_params),
                                             'toe')
+            Flipper.enable(:toe_light_house_dgi_direct_deposit)
+
+            expect(response.status).to eq(200)
+          end
+        end
+      end
```

Comment:
Please include tests for when the features is disabled. This is the suggested format

```ruby
  context "Feature feature_flag=true" do
    before do
      Flipper.enable(:feature_flag)
    end
    it "behaves this way with the feature enabled" do
    end
  end
  context "Feature feature_flag=false" do
    before do
      Flipper.disable(:feature_flag)
    end
    it "behaves this way with the feature disabled" do
    end
  end 
```